.github/workflows/build_main_documentation.yml

```diff
@@ -95,15 +99,22 @@ jobs:
           cp doc-build/${{ inputs.package }}/_versions.yml build_dir/${{ inputs.package }}
 
       - name: Make documentation
+        shell: bash
         env:
           NODE_OPTIONS: --max-old-space-size=6656
         run: |
           echo "doc_folder has been set to ${{ env.doc_folder }}"
 
           if [ -z "${{ inputs.notebook_folder }}" ]
           then
-            cd doc-builder &&
-            doc-builder build ${{ inputs.package }} ../${{ env.doc_folder }} --build_dir ../build_dir --clean --html ${{ inputs.additional_args }} &&
+            IFS=', ' read -r -a langs <<< "${{ inputs.languages }}"
+
+            cd doc-builder
+            for lang in "${langs[@]}"
+            do
+                echo "Generating docs for language $lang"
+                doc-builder build ${{ inputs.package }} ../${{ env.doc_folder }}/$lang --build_dir ../build_dir --clean --html ${{ inputs.additional_args }} --language $lang
```

Comment:
This breaks all the documentations of the libraries, where the source files are in docs/source/, which doesn't end with `en`, so we can't make that change without a broader concertation with the other opensource teams.

Reply:
Indeed, that's what I observed with the way the Spanish translation is currently implemented in the `transformers` docs. 

An alternative would be to provide a mapping between the language and the path where the docs live, e.g. in `transformers` this could be:

```
{"en":"path/to/docs/source", "es":"path/to/docs/source_es"}
```

But that's kind of clunky because we also have args like `path_to_docs` which is used to set `env.doc_folder`. Did you already have something in mind for handling this in `transformers`?

Reply:
We can agree to use that format ultimately (have a folder per language in `docs/source`, I think it makes sense. But it should be opt-in so here `lang` should default to an empty string and when it's an empty string, we should launch the same command as before. This way when a given repo wants to switch to building several languages in the doc, they can do so by reorganizaing the doc folder and providing the argument to the doc building job.

Does that make sense?

Reply:
I like your idea to opt-in! Will adjust the PR accordingly :)